packages/core-browser/src/components/actions/index.tsx

```diff
@@ -64,7 +64,7 @@ export const MenuActionList: React.FC<{
       }
 
       // hacky: read MenuNode from MenuItem.children.props
-      const menuItem = item.props.children.props.data as MenuNode;
+      const menuItem = item.props.children[0].props.data as MenuNode;
```

Comment:
You can make a breakpoint to see why.
Item.props.children is an array with two elements and the first element contains props.data.